ztp/source-crs-builder/MachineConfigChronyDynamicMasterParams/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+GOPATH=${GOPATH:-${HOME}/go}
+GOBIN=${GOBIN:-${GOPATH}/bin}
+MCMAKER=${MCMAKER:-${GOBIN}/mcmaker}
+
+${MCMAKER} -stdout -name 05-chronyd-dynamic-params -mcp master \
+	dropin -source 20-environment-vars.conf -for chronyd-restart.service
```

Comment:
I had a question about this.  If eventually you're going to want the drop-in in the same MC object as the service, why have a drop-in at all?  Just put these environment variables right in the service.  Admins could craft custom drop-ins to change the values, but we don't need to do that since we own the original .service file.

Reply:
Agree, removing the dropin